src/styles/main.scss

```diff
@@ -1,3 +1,70 @@
-body {
-  margin: 0;
+$cell-size: 100px;
+$border-color: black;
+$border-width: 1px;
+$number-size: 30px;
+$gap: 1px;
+$padding: 10px;
+$days-in-week: 7;
+$animation-duration: 0.5s;
+$start-days: mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6;
+
+.main-container {
+  display: flex;
+  justify-content: center;
+}
+
+.calendar {
```

Comment:
Divide each BEM block to separate files. Also variables move to separate file too